test/Spec/BotPlutusInterface/Balance.hs

```diff
@@ -97,11 +87,8 @@ addUtxosForChange = do
       minUtxo = [(txout, 1_000_000)]
       fees = 500_000
       utxoIndex = Map.fromList [utxo1, utxo2, utxo3]
-      privKeys = Map.fromList [(pkh1, privateKey1)]
-      requiredSigs = [pkh1]
       ownPkh = pkh1
-      pparams = def {protocolParamUTxOCostPerWord = Just (Lovelace 1)}
-      prebalancedTx =
-        PreBalance.preBalanceTx pparams minUtxo fees utxoIndex ownPkh privKeys requiredSigs tx
+      balancedTx =
+        Balance.balanceTxStep minUtxo fees utxoIndex ownPkh tx
 
-  txInputs <$> prebalancedTx @?= Right (Set.fromList [txIn1, txIn2, txIn3])
+  txInputs <$> balancedTx @?= Right (Set.fromList [txIn1, txIn2])
```

Comment:
Note this input was removed, unsure why it was ever there?
The expected output of this tx is 1.6ADA + 0.5ADA fee, inputs 1 and 2 are 1.1ADA and 1.0ADA, both summing to 2.1ADA, so only 2 inputs should be needed to cover it.

Reply:
I think this is the edge case we talked about, when even though the inputs and outputs are balanced, the cli adds a change utxo which must be at least the minUtxo (around 1 ada). This is now outdated.
I'm wondering though what happens if we change the amout being sent to 1.5, so the whole amount is 2ADA, with a change of 0.1, which then must be rounded up.